dagrr/fly.go

```diff
@@ -67,7 +101,15 @@ kill_timeout = 30
 
 [[vm]]
   size = "%s"
-	`, m.Dagrr.App, m.Dagrr.Version, disk, size)
+`, m.Dagrr.App, primaryRegion, envVars, m.Dagrr.Version, disk, size)
```

Comment:
I *think* this is setting up the wrong engine image. We need the `-gpu` variant suffixed to `m.Dagrr.Version`:

```suggestion
`, m.Dagrr.App, primaryRegion, envVars, m.Dagrr.Version + "-gpu", disk, size)
```

As suggested by @gerhard in https://github.com/dagger/dagger/issues/9015#issuecomment-2490312347.

Reply:
I added this change, but only when setting a "gpu kind". We want to standard engine image to be used when no need for a gpu.